filters/filters.go

```diff
@@ -38,10 +38,13 @@ type FilterContext interface {
 	// route.
 	Served() bool
```

Comment:
If there is an impliciit way to determine the state of this, we should use it. Otherwise it is unnecessary data-denormalisation and will only lead to bugs in the long run